### ISSUE TITLE
Java Script basics and advance

### ISSUE THREAD

**Issue by Himanshuvashist** [CodingGarden/landscaping#13]
Your idea may have already been suggested! Please search on the [issues tracker](https://github.com/CodingGarden/landscaping/issues) before creating a new one.

>NOTE: Please give your issue a descriptive title so it is easy to identify at a glance.

## (REQUIRED) What category does this issue fit into?

* [ ] App Idea
* [ ] App Constraint
* [ ] Front-end Technology
* [ ] Back-end Technology
* [ ] Deployment
* [ ] DevOps
* [ ] Code Craft
* [ ] Workflow/Tools
* [x] Show Type or Series
* [ ] Something Else

## (REQUIRED) How are you affiliated with Coding Garden?

* [x] [YouTube Subscriber](https://www.youtube.com/codinggardenwithcj?sub_confirmation=1)
* [ ] [Patron](https://www.patreon.com/CodingGardenWithCJ)
* [ ] [Donor](https://streamlabs.com/CodingGardenWithCJ)

## (REQUIRED) Describe your suggestion

>Make video on Jave script so that newb like me can understand your coding properly and your videos on React and vue or any other technologies related to JS. Cover advance topics as well like ECMA script and why they are importand and stufflike that. Java Script syntax part is easy but i am having dificulty in advance topics like understanding ECMA script ,currying(advance function in JS) and many more things like that. 
So in short i want you to make series of JS where we can learn whole JS.
